test/utils/fizzy_engine.cpp

```diff
@@ -72,9 +73,48 @@ bytes_view FizzyEngine::get_memory() const
     return {m_instance->memory->data(), m_instance->memory->size()};
 }
 
-std::optional<WasmEngine::FuncRef> FizzyEngine::find_function(std::string_view name) const
+namespace
 {
-    return fizzy::find_exported_function(m_instance->module, name);
+bool matching_signature(const fizzy::FuncType& func_type, std::string_view signature)
```

Comment:
Probably would be nicer to have simple translator instead (sig -> func_type) and equal operator for `func_type`.